src/ln/msgs.rs

```diff
@@ -86,8 +87,12 @@ impl LocalFeatures {
 	pub(crate) fn supports_upfront_shutdown_script(&self) -> bool {
 		self.flags.len() > 0 && (self.flags[0] & (3 << 4)) != 0
 	}
-	pub(crate) fn requires_upfront_shutdown_script(&self) -> bool {
-		self.flags.len() > 0 && (self.flags[0] & (1 << 4)) != 0
+	pub(crate) fn set_upfront_shutdown_script(&mut self) {
```

Comment:
Why is this a method? We always set it so make it the return value in new(). If some client needs it in the future we can make an unset_ function.

Reply:
Good point, corrected